Website/index.html

```diff
@@ -15,12 +15,12 @@
             <img src="assets/images/logo.png">
             <h2>Hypixel API</h2>
             <span>
-                <a class="button" href="https://github.com/HypixelDev/PublicAPI">GitHub Repo</a>
```

Comment:
"Repo" is a very widely used abbreviation for "repository"

Reply:
alright